docs/dev_guide/NonContainerizedDeveloperCustomizations.md

```diff
@@ -0,0 +1,64 @@
+# Non Containerized NSFS - Developer customization (OPTIONAL)
+
+The following list consists of supported optional developer customization  -
+
+## 1. Number of forks - 
+ 
+**Decription -** Adjust the number of forks in order to increase the S3 endpoints number. This will enable NooBaa handle a higher scale of S3 requests concurrently.
```

Comment:
```suggestion
**Description -** Adjust the number of forks in order to increase the S3 endpoints number. This will enable NooBaa to handle a higher scale of S3 requests concurrently.
```